static/src/javascripts/lib/capture-ophan-info.js

```diff
@@ -38,9 +39,18 @@ const capturePerfTimings = (): void => {
         })),
     };
 
+    let ophanInfo = {};
+
+    if (config.get('page.dcrCouldRender', false)) {
+        ophanInfo = { ...ophanInfo, ...{ experiences: 'dcrCouldRender' } };
```

Comment:
As experiences is an array in the DL, how does this map?

Reply:
Splits on comma https://github.com/guardian/ophan/pull/3427/files#diff-33277ec3db9ad2eab2030481b9aa5842R42